src/components/input.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const Input = () => {
+    return(
+        <div>
```

Comment:
This is meant to be a reusable component but right now it's always going to give us 2 inputs